README.md

```diff
@@ -31,7 +31,7 @@ var parseChangelog = require('changelog-parser')
 If provided with a callback, `parseChangelog` will invoke the function with the parsed changelog.
 
 ```js
-parseChangelog('path/to/CHANGELOG.md', function (err, result) {
+parseChangelog('path/to/CHANGELOG.md', null, function (err, result) {
```

Comment:
I'm realizing this introduces a bigger problem, which is that this is a breaking change for current users of the node API.

One way to get around this if we want to introduce this in a minor release is to make the function [variadic](https://en.wikipedia.org/wiki/Variadic_function), allowing for the options object to be optional.

This can be achieved like so:

```js
function parseChangelog (file, options, callback) {
  if (typeof callback === 'undefined') {
    callback = options
    options = {}
  }
```

If you're willing to make one more change I think this would be ready to be merged. Thanks for being patient and working through this with me 😄

Reply:
That's ok. I'm the one who have to thank for such quick replies and will to get this in.

I completely understand you want to avoid a major release for this, so I submitted the requested changes.